src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -1921,7 +2006,31 @@ void testDcBusBarSectionContingency() {
     }
 
     @Test
-    void testBusBarSectionContingencyWithSlackBusRelocation() {
+    void testDcBusBarSectionContingencyFastMode() {
```

Comment:
This test should be reworked if contingencies on switches are handled in this PR.

Reply:
why testDcBusBarSectionContingency isn't a parametrized test with fastMode true/false ?

Reply:
Currently contingencies on switches are not handled, but we are working on it. When done, this test will be parametrized.

Reply:
Contingencies are now handled, and the TU is parameterized. Thank you for the comments. I resolve this conversation.